x-pack/plugins/security_solution/public/siem_migrations/rules/components/rule_details_flyout/tabs/summary/index.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { EuiCommentProps } from '@elastic/eui';
+import { EuiAvatar, EuiCommentList, EuiMarkdownFormat, EuiSpacer } from '@elastic/eui';
+import moment from 'moment';
+import { AssistantAvatar } from '@kbn/elastic-assistant';
+import { useTimeZone } from '../../../../../../common/lib/kibana';
+import {
+  RuleMigrationStatusEnum,
+  type RuleMigration,
+} from '../../../../../../../common/siem_migrations/model/rule_migration.gen';
+import * as i18n from './translations';
+
+interface SummaryTabProps {
+  ruleMigration: RuleMigration;
+}
+
+export const SummaryTab: React.FC<SummaryTabProps> = React.memo(({ ruleMigration }) => {
+  const timezone = useTimeZone();
+  const timestamp = useMemo(
+    // Date formats https://momentjs.com/docs/#/displaying/format/
+    () => moment.tz(ruleMigration['@timestamp'], timezone).format('ll'),
```

Comment:
double check tz please

Reply:
Yeah, you were right. We don't really need it here. Removed tz usage